docker/DockerFilePyneUbuntu1704Python2

```diff
@@ -0,0 +1,64 @@
+#
+# Ubuntu Dockerfile
+#
+# https://github.com/dockerfile/ubuntu
+#
+
+# pull the base image
+FROM ubuntu:17.04
+
+# install
+RUN \
+    apt-get -y update && \ 
+    apt-get -y install gcc g++ make openssl
+RUN \
+    apt-get -y install cmake curl git htop man unzip vim wget emacs
+RUN \ 
+    apt-get -y install libhdf5-dev libhdf5-100 gfortran && \
+    apt-get -y install expat libtool autoconf && \
+    apt-get -y install patch texinfo && \
+    apt-get -y install dejagnu bison flex && \
+    apt-get -y install libblas-dev liblapack-dev && \
+    apt-get -y install python && \
+    apt-get -y install subversion python-dev python-pip
+# install moab
+RUN \
+    echo "#!/bin/bash" > /opt/env.sh && \
+    cd /opt && \
+    mkdir moab && \
+    git clone https://bitbucket.org/fathomteam/moab && \
+    cd moab && \
+    git checkout 4.9.1 && \
+    autoreconf -fi && \
+    mkdir bld && \
+    cd bld && \
+    ../configure --enable-optimize --enable-shared --enable-dagmc --with-hdf5=/usr/lib/x86_64-linux-gnu/hdf5/serial/ --prefix=/opt/moab/ && \
+    make -j4 && \ 
+    make check && \ 
+    make install && \
+    cd .. && \ 
+    rm -rf bld moab
```

Comment:
If this is attempting to delete the `moab` that is the repo, then I think you're in the wrong directory.  
You `cd ..` from `/opt/moab/moab/bld`